README.md

```diff
@@ -17,8 +17,8 @@ There are two recommended ways of running the project:
 
 ### Running with Docker
 Using Docker has the advantage of starting everything up automatically,
-including certificate generation, tests and linting, as well as creating
-both the SECC and EVCC.
+including certificate generation, tests and linting, as well as spawning
```

Comment:
Either is fine in this context, I think -- @nikkigithub1 was there a specific reason to change "spawning" to "creating" in the first place?